src/simulation/m_global_parameters.fpp

```diff
@@ -838,13 +851,26 @@ contains
                     end if
                 end if
 
-                if (hypoelasticity) then
+                if (hypoelasticity .or. hyperelasticity) then
+                    elasticity = .true.
```

Comment:
i prefer making `elasticity` an input parameter that the user has to specify in `case.py`. and then if the user tries to use an elasticity feature but elasticity is false, then it fails. since you are including the scaffolding for hyper and hypo, you may need two such parameters set by the user.